### DISCUSSION TITLE
Kratos and Bandwith

### DISCUSSION THREAD

**Discussion by rbillig** [ory/kratos#2955] [Q&A] [answered]
Hello,

My Ory Kratos work with an remote database server.
When digging, I can see that the communication between kratos and the database takes around 4kb/s permanent bandwith (250 Mo/hour all day long).

Is this something normal ? Is there a way to reduce this ?

Cdlt
RB

**Comment by vinckr** [comment]
Hey, 
thanks for the report.
I don't think it is normal, but hard to say for me what is the issue without a detailed look at your system, what database are you running, at what load, in what way is Ory Kratos configured & used, etc.

**Reply by rbillig** [reply]
Hello,
Thanks for the answer.
Basically, this is a simple kratos installation with just me and myself as end user  and I recently migrate the database from a local Postgrey to a MariaDB (MySql) remote server.

There is no real issue here but 4Kb/s permanent communication between the kratos server and the database seems unusual when you consider kratos has nothing to work on.

If someone has any hint on how to reduce this, I'm listening.

Regards
RB

**Comment by rbillig** [comment, marked as answer]
After Digging a little bit more, it seems, the option :  --watch-courier is the source of the network bandwith usage (between Kratos and the remote database).

**Reply by vinckr** [reply]
Ah, that makes sense! 
Glad you figured it out :-)

**Reply by rbillig** [reply]
But I would like to reduce this and I can't see any appropriate setting. 

In the documentation, I can see there are 4 database check per seconds regarding mail. If the need is just to certify/verify an Email address, Maybe an attempt every 5 seconds is enough (my case) ...
